### PR TITLE
Added standard material sample of texture based alpha testing

### PR DESCRIPTION
**Pull request by mvaligursky**
Added this to standard material API doc:

Screen Shot 2021-02-23 at 10 43 12 AM

### CODE REVIEW

**Review by raytranuk — APPROVED** [review]
Looks useful

File: src/scene/materials/standard-material.js [hunk 1]
```diff
@@ -324,6 +324,20 @@ class Chunks {
...
+ * @example
+ * // Create a new Standard material
+ * var material = new pc.StandardMaterial();
+ *
+ * // assign a texture to diffuse slot
+ * material.diffuseMap = texture;
+ *
+ * // use the alpha channel of the texture for alpha testing with a reference value of 0.5
+ * material.opacityMap = texture;
+ * material.opacityMapChannel = "a";
```
Comment (willeastcott) [inline src/scene/materials/standard-material.js:336]: [This line](https://github.com/playcanvas/engine/blob/master/src/scene/materials/standard-material.js#L1134) seems to indicate that this _should_ be the default. Although perhaps there's a bug.

Comment (willeastcott) [reply src/scene/materials/standard-material.js:336]: Ideally, the default should be documented [here](https://github.com/playcanvas/engine/blob/master/src/scene/materials/standard-material.js#L167).

Comment (willeastcott) [reply src/scene/materials/standard-material.js:336]: Confirmed:

image

**Review by willeastcott — APPROVED** [review]

**Pull request merged**